library/src/main/java/com/midtrans/httpclient/APIHttpClient.java

```diff
@@ -116,12 +121,12 @@ private static OkHttpClient buildHttpClient(Config config) {
         // setup default http client
         if (config.getProxyConfig() == null) {
             return httpClient.newBuilder()
-                    .addInterceptor(loggingInterceptor(config))
                     .connectTimeout(config.getConnectionTimeout(), config.getHttpClientTimeUnit())
                     .readTimeout(config.getReadTimeout(), config.getHttpClientTimeUnit())
                     .writeTimeout(config.getWriteTimeout(), config.getHttpClientTimeUnit())
                     .connectionPool(connectionPool)
                     .addInterceptor(headersInterceptor)
+                    .addInterceptor(loggingInterceptor(config))
```

Comment:
There is 2 `addInterceptor` here, will the bottom overwrite the top one? or both will be added?
If it get overwritten, then it seems like a potential bug

Reply:
Yes, that's expected, both will be added @rizdaprasetya

Reply:
cool, that will be fine 👌